### PR TITLE
Switch to shakapaker

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: bin/webpacker [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
```
Comment (marlo-longley) [inline bin/webpacker:1]: @jcoyne this file has very similar contents to bin/webpack-dev-server, except for a different runner. What is its purpose?

Comment (jcoyne) [reply bin/webpacker:1]: It builds the javascript. See https://github.com/shakacode/shakapacker#development

File: bin/webpacker-dev-server [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
```
[the remark this answers is not in the document]
Comment (jcoyne) [reply bin/webpacker-dev-server:1]: @marlo-longley hasn't that already been done? You're pointing at a commit not in this branch.

**Review by corylown — COMMENTED** [review]
This could just be an issue with my local setup or maybe it's expected, but I had to run `yarn add @blueprintjs/core` and `yarn add @blueprintjs/icons` to get `bin/webpacker-dev-server` to start without errors. I was seeing things like: `Module not found: Error: Can't resolve '@blueprintjs/core' in '/Users/corylown/workspace/sul-embed/node_modules/react-mosaic-component/lib/util'`

Update: not harmful and due to blueprintjs being an optional dependency

**Review by corylown — APPROVED** [review]

**Pull request merged**